src/components/snackbar/snackbar.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import { useState, forwardRef, useImperativeHandle } from "react"
+import "./snackbar.css"
+
+export const Snackbar = forwardRef((props, ref) => {
+  const [showSnackbar, setShowSnackbar] = useState(false);
+
+  useImperativeHandle(ref, () => ({
+    show() {
+      setShowSnackbar(true);
+      setTimeout(() => {
+        setShowSnackbar(false);
+      }, 2000);
+    },
+  }))
+  
+  return (
+  <div className="snackbar" id={showSnackbar ? "show" : "hide"}>
```

Comment:
I would usually prefer to do this with [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes). Shifting the ID dynamically can have unexpected consequences for tools that rely on them being consistent. 

I don't think you'd run into that situation here but it's a good practice to get into!

So you would set something like `data-state="visible"` then you can use that in your CSS to control the visibility.

Reply:
Nice! I've made that change. Works a treat.